From ecada2657b9552e9743c38ae8d5fa508a75b2aca Mon Sep 17 00:00:00 2001
From: "tpopela@redhat.com"
 <tpopela@redhat.com@268f45cc-cd09-0410-ab3c-d52691b4dbfc>
Date: Wed, 1 Mar 2017 14:14:04 +0000
Subject: [PATCH 1/3] [GTK] Fix problems found by Coverity scan in
 WebPrintOperationGtk.cpp https://bugs.webkit.org/show_bug.cgi?id=169027

Reviewed by Carlos Garcia Campos.

Initialize the lastPagePosition variable and remove an unused one
- totalToPrint.

* WebProcess/WebPage/gtk/WebPrintOperationGtk.cpp:
(WebKit::PrintPagesData::PrintPagesData):

git-svn-id: http://svn.webkit.org/repository/webkit/trunk@213217 268f45cc-cd09-0410-ab3c-d52691b4dbfc
---
 .../WebKit2/WebProcess/WebPage/gtk/WebPrintOperationGtk.cpp |  2 +-
 2 files changed, 14 insertions(+), 1 deletion(-)

diff --git a/Source/WebKit2/WebProcess/WebPage/gtk/WebPrintOperationGtk.cpp b/Source/WebKit2/WebProcess/WebPage/gtk/WebPrintOperationGtk.cpp
index b6205db..fdd4193 100644
--- a/Source/WebKit2/WebProcess/WebPage/gtk/WebPrintOperationGtk.cpp
+++ b/Source/WebKit2/WebProcess/WebPage/gtk/WebPrintOperationGtk.cpp
@@ -210,6 +210,7 @@ struct PrintPagesData {
         , firstSheetNumber(0)
         , numberOfSheets(0)
         , firstPagePosition(0)
+        , lastPagePosition(0)
         , collated(0)
         , uncollated(0)
         , isDone(false)
@@ -362,7 +363,6 @@ struct PrintPagesData {
     GRefPtr<GMainLoop> mainLoop;
 
     int totalPrinted;
-    size_t totalToPrint;
     int pageNumber;
     Vector<size_t> pages;
     size_t sheetNumber;
-- 
2.9.3

